### PR TITLE
M0090IDE-4478 Add a tutorial on how to create a branded Eclipse

### PR DESCRIPTION
**Pull request by tdelhomenie-microej**
https://docs.microej.com/en/feature-m0090ide-4478_branded-eclipse/SDK6UserGuide/tutorials/brandEclipse.html

### CODE REVIEW

File: SDK6UserGuide/tutorials/brandEclipse.rst [hunk 1]
```diff
@@ -0,0 +1,194 @@
+.. _sdk_6_brand_eclipse:
+
+How To Brand an Eclipse IDE
```
[the remark this answers is not in the document]
Comment (tdelhomenie-microej) [reply SDK6UserGuide/tutorials/brandEclipse.rst]: Done.

File: SDK6UserGuide/tutorials/brandEclipse.rst [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+This can be very convenient if you need to redistribute your own unique Eclise IDE version, customized to your brand.
+
+This tutorial will guide you through the steps to create such a branded Eclipse IDE.
+
+Install Eclipse and the MicroEJ Plugin
+--------------------------------------
+
+- Download the `Eclipse IDE for RCP and RAP Developers Edition <https://www.eclipse.org/downloads/packages/release/2023-09/r/eclipse-ide-rcp-and-rap-developers>`__.
+- Install and launch it.
+- Install the Microej plugin as described in the ``Eclipse`` tab of the :ref:`sdk_6_install_ide_plugin` chapter.
```
[the remark this answers is not in the document]
Comment (tdelhomenie-microej) [reply SDK6UserGuide/tutorials/brandEclipse.rst]: Done.

File: SDK6UserGuide/tutorials/brandEclipse.rst [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+.. figure:: images/eclipse-branding-create-project.png
+    :alt: Creation of a Branding Eclipse Project
+    :align: center
+    :scale: 70%
+
+    Creation of a Branding Eclipse Project
+
+- Click on the :guilabel:`Finish` button.
+
+Copy the branding resources
```
[the remark this answers is not in the document]
Comment (tdelhomenie-microej) [reply SDK6UserGuide/tutorials/brandEclipse.rst]: Done.

File: SDK6UserGuide/tutorials/brandEclipse.rst [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+- Copy the image file(s) of the IDE launcher in the ``images`` folder. 
+  The image format depends on the Operating System: 
+  
+    - ``icon.xpm`` for Linux
+    - ``icon.icns`` for macOS
+    - ``icon.ico`` file or ``icon.bmp`` files for Windows. 
+      For Windows, if bmp files are used, it is required to provide one bmp file for each one of the following resolutions: 16x16, 32x32, 64x64, 128x128, 256x256.
+
+- Copy the image files associated with the Application window (visible in the Windows dock for example) in the ``images`` folder. 
+  The image format must be bmp, with one bmp file for each one of the following resolutions: 16x16, 32x32, 64x64, 128x128, 256x256.
```
Comment (bguedas) [inline SDK6UserGuide/tutorials/brandEclipse.rst:58]:
```suggestion
  The image format must be png, with one png file for each one of the following resolutions: 16x16, 32x32, 48x48, 64x64, 128x128, 256x256.
```

File: SDK6UserGuide/tutorials/brandEclipse.rst [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+  - ``META-INF`` folder
+  - ``plugin.xml`` file
+  - ``splash.bmp`` file
+  - ``images`` folder
+  - ``plugin_customization.ini`` file (if exists)
+
+- Save your changes in the ``build.properties`` file.
+
+- Right-click on the project, then click on :guilabel:`Export...`.
+- Select :guilabel:`Plug-in Developement` > :guilabel:`Eclipse product`, then click on the guilabel:`Next` button.
```
Comment (bguedas) [inline SDK6UserGuide/tutorials/brandEclipse.rst:181]:
```suggestion
- Select :guilabel:`Plug-in Developement` > :guilabel:`Eclipse product`, then click on the :guilabel:`Next` button.
```

**Review by bguedas — CHANGES_REQUESTED** [review]
We should add recommended sizes for the images. The "Branding" tab says that the image should not exceed 500x330 and that the text is not shown if it exceeds 250x330.

File: SDK6UserGuide/tutorials/brandEclipse.rst [hunk 6]
```diff
@@ -0,0 +1,194 @@
...
+
+- Copy the image file of the splash screen at the root of the project. The following name and image types are supported:
+
+  - ``splash.png``
+  - ``splash.jpg``
+  - ``splash.jpeg``
+  - ``splash.gif``
+  - ``splash.bmp``
+
+- Create a folder named ``images``.
```
[the remark this answers is not in the document]
Comment (tdelhomenie-microej) [reply SDK6UserGuide/tutorials/brandEclipse.rst]: Done.

File: SDK6UserGuide/tutorials/brandEclipse.rst [hunk 7]
```diff
@@ -0,0 +1,194 @@
...
+  The image format depends on the Operating System: 
+  
+    - ``icon.xpm`` for Linux
+    - ``icon.icns`` for macOS
+    - ``icon.ico`` file or ``icon.bmp`` files for Windows. 
+      For Windows, if bmp files are used, it is required to provide one bmp file for each one of the following resolutions: 16x16, 32x32, 64x64, 128x128, 256x256.
+
+- Copy the image files associated with the Application window (visible in the Windows dock for example) in the ``images`` folder. 
+  The image format must be bmp, with one bmp file for each one of the following resolutions: 16x16, 32x32, 64x64, 128x128, 256x256.
+- Copy the ``About`` dialog (:guilabel:`Help` > :guilabel:`About...`) image in the ``images`` folder.
```
[the remark this answers is not in the document]
Comment (tdelhomenie-microej) [reply SDK6UserGuide/tutorials/brandEclipse.rst]: Done.

File: SDK6UserGuide/tutorials/brandEclipse.rst (hunk 4 above)
Comment (tdelhomenie-microej) [reply SDK6UserGuide/tutorials/brandEclipse.rst]: Done.

File: SDK6UserGuide/tutorials/brandEclipse.rst (hunk 5 above)
Comment (tdelhomenie-microej) [reply SDK6UserGuide/tutorials/brandEclipse.rst]: Done.

**Comment by tdelhomenie-microej** [comment]
@bguedas I refactored the branding resources copy by dispatching in each section. Let me know what you think.

**Review by bguedas — APPROVED** [review]
LGTM, it should be reviewed by IoT team now.

**Comment by tdelhomenie-microej** [comment]
@abaloche-microej do you confirm you did a review and a successful test?

**Pull request merged**